iml-api/src/graphql.rs

```diff
@@ -80,6 +80,30 @@ struct TargetResource {
     cluster_hosts: Vec<i32>,
 }
 
+struct BannedTargetResource {
+    resource: String,
+    cluster_id: i32,
+    host_id: i32,
+    mount_point: Option<String>,
+}
+
+#[derive(juniper::GraphQLObject)]
+/// A Corosync banned resource
+struct BannedResource {
+    // The resource id
```

Comment:
These should be doc comments 

```suggestion
    /// The resource id
```